docs/about-filecoin/wrapped-filecoin.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Wrapped Filecoin
+description: Wrapped tokens allow tokens from one blockchain to be used natively on another chain. 
+---
+
+# Wrapped Filecoin
+
+## What is a wrapped token?
+
+Wrapped tokens allow tokens from one blockchain to be used natively on another chain. The wrapping process usually involves locking the original tokens in a smart contract, which then mints an equivalent amount of wrapped tokens on another blockchain. 
+
+Users can transact confidently because the wrapped token value is preserved by a smart contract that backs each token one-to-one with the underlying assets. This gives users greater flexibility — such as the ability to write smart contracts with assets across multiple networks without waiting for both chains to update, to create greater liquidity for decentralized finance, or to optimize block confirmation times or gas fees by choosing different networks.
+
+::: callout
+Although Filecoin tokens (FIL) cannot be directly used on Ethereum (ETH) or other networks due to different standards, **wrapped FIL** offer the ability to use FIL tokens 1:1 within other blockchain networks including Ethereum, Polygon, Solana, Celo, and more.
+:::
+
+## Where can I use wrapped FIL?
+
+Wrapped FIL is available in the following networks:
+
+- Ethereum (renFIL, wFIL)
+- Binance Smart Chain (renFIL, BFIL)
+- Fantom (renFIL)
+- Polygon (renFIL)
+- Avalanche (renFIL)
+- Solana (renFIL)
+- Arbitrum (renFIL)
+- Celo (wFIL)
+- STACKS (coming soon wFIL)
+- KADENA (coming soon wFIL)
+- etc.
+
+## Custodial or non-custodial wrapping solution?
+
+- **Non-custodial designs** (renFIL, vFIL) are governed only by on-chain, immutable smart contracts. No single or collective entity has a master key that could force trades, transactions, or other operations to follow one direction or the other.
+- **Custodial designs** (wFIL, BFIL, eFIL, HFIL) are managed by a single third party. They may have services always have a counter-party risk from their operator but could have some other advantages like insurance, guaranteed backing, support, etc.
+
+## Wrapping your FIL process
```

Comment:
```suggestion
## How to wrap your FIL
```

Reply:
```suggestion
## How to wrap your FIL?
```

Sounds good to me. 
@mishmosh what do you think?